connector_search_engine/models/se_binding.py

```diff
@@ -103,14 +105,60 @@ def unlink(self):
             self.env["se.binding.todelete"].sudo().create(todelete_vals_list)
         return super().unlink()
 
-    def jobify_recompute_json(self, force_export=False):
-        description = _("Recompute %s json and check if need update" % self._name)
+    def _get_binding_to_process(self, bindings, batch_size):
+        return bindings[0:batch_size], bindings[batch_size:]  # noqa: E203
+
+    def _jobify_get_job_description(self, processing_count):
+        # We check if we are currently handling an exception in order
+        # to change the job description.
+        (current_exception, current_exception_value, __) = sys.exc_info()
+        if current_exception:
+            return _(
+                "Batch of %(processing_count)s items for %(model_name)s. "
+                "Previous batch raised %(exception_name)s: %(exception_value)s."
+            ) % dict(
+                processing_count=processing_count,
+                model_name=self._name,
+                exception_name=current_exception.__name__,
+                exception_value=current_exception_value,
+            )
+        else:
+            return _(
+                "Batch task of %(processing_count)s for "
+                "recomputing %(model_name)s json"
+            ) % dict(
+                processing_count=processing_count,
+                model_name=self._name,
+            )
+
+    def jobify_recompute_json(self, force_export=False, batch_size=500):
         # The job creation with tracking is very costly. So disable it.
-        for record in self.with_context(tracking_disable=True):
-            record.with_delay(description=description).recompute_json(
+        bindings = self.with_context(tracking_disable=True)
+        while bindings:
+            processing, bindings = self._get_binding_to_process(bindings, batch_size)
+            description = self._jobify_get_job_description(len(processing))
```

Comment:
```suggestion
            description = self._jobify_get_job_description(processing)
```
and get the count inside that method